study.md

```diff
@@ -160,15 +163,20 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+An answer from a server based on the request being made. Response codes include:
+1xx: waiting
+2xx: a response including the result you requested
+3xx: go away
+4xx: something wrong with request
+5xx: somethine wrong on server, unable to process (internal error)
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl --request GET https://www.twitter.com
```

Comment:
This is a correctly formatted curl request but I don't think it will go to reddit.com !